libs/blocks/merch-card/merch-card.js

```diff
@@ -102,6 +102,15 @@ function addMerchCardGridsIfMissing(section) {
   }
 }
 
+const decorateMerchCardLinkAnalytics = (el) => {
+  const links = el.querySelectorAll('a');
```

Comment:
nit: no need for a constant, you can `[...el.querySelectorAll('a')].forEach` directly